### PR TITLE
fix(slider): decimal and small values

### PR DESCRIPTION
**Pull request by YonathanMeguira**
fix #160 
Fix for small values, new branch containing only the changes for dealing with small values + corrected missing control in multi-slider.example

### CODE REVIEW

File: projects/datorama/core/src/lib/slider/slider.component.ts [hunk 1]
```diff
@@ -496,13 +477,17 @@ export class DatoSliderComponent extends BaseCustomControl implements AfterViewI
...
-    let minDistance = this.model.min - this.options.min;
-    let maxDistance = this.model.max - this.options.min;
-    minDistance *= this.unitSize;
-    maxDistance *= this.unitSize;
+    if (!this.multiCursor) {
+      this.model.min = this.options.min;
+    }
+    const howManySteps = this.sliderRange / this.options.step;
+    this.unitSize = this.sliderDimensions.width / howManySteps;
+    const minCursorX = ((this.model.min - this.options.min) / this.sliderRange) * this.sliderDimensions.width;
```
Comment (NetanelBasal) [inline projects/datorama/core/src/lib/slider/slider.component.ts]:
```ts
const calcX = value => ((value - this.options.min) / this.sliderRange) * this.sliderDimensions.width;
const minCursorX = calcX(this.model.min);
const maxCursorX = calcX(this.model.max);
```

Comment (YonathanMeguira) [reply projects/datorama/core/src/lib/slider/slider.component.ts]: I only calculate it once for minCursorX, not maxCursorX

Comment (NetanelBasal) [reply projects/datorama/core/src/lib/slider/slider.component.ts]: I see twice:
```ts
const minCursorX = ((this.model.min - this.options.min) / this.sliderRange) * this.sliderDimensions.width;
 const maxCursorX = ((this.model.max - this.options.min) / this.sliderRange) * this.sliderDimensions.width;
```

Comment (YonathanMeguira) [reply projects/datorama/core/src/lib/slider/slider.component.ts]: saw it now, replacing it

Comment (YonathanMeguira) [reply projects/datorama/core/src/lib/slider/slider.component.ts]: fixed

Comment (NetanelBasal) [reply projects/datorama/core/src/lib/slider/slider.component.ts]: Next time, please use `git commit --amend` as I wrote in the docs.

**Pull request merged**